packages/react/src/TreeView/TreeView.tsx

```diff
@@ -646,6 +646,11 @@ const SubTree: React.FC<TreeViewSubTreeProps> = ({count, state, children}) => {
 
         setLoadingFocused(false)
       }
+    } else if (state === 'loading') {
+      if (!parentElement) return
```

Comment:
nit: could bubble up as well and return early on 619, since other if statement checks and returns for same condition on 6221

Reply:
I see! So adding `if (!parentElement) return` at the top rather than in both conditionals. Thank you for the suggestion!